src/Adapters/BlobAdapterUtility.ts

```diff
@@ -232,3 +237,23 @@ function sendConfigTelemetry(data: I3DScenesConfig) {
         })
     );
 }
+
+/** Helper function to handle all schema migration transformations */
+export function handleMigrations(data: I3DScenesConfig) {
```

Comment:
As a rule I like this as a way to immediately forward compat old schemas, cool pattern!